packages/sdk/src/client.ts

```diff
@@ -353,12 +361,12 @@ export class AcrossClient {
    * @returns See {@link GetAvailableRoutesReturnType}.
    */
   async getAvailableRoutes(
-    params: Omit<GetAvailableRoutesParams, "apiUrl" | "logger">,
+    params: MakeOptional<GetAvailableRoutesParams, "apiUrl" | "logger">,
   ): Promise<GetAvailableRoutesReturnType> {
     return getAvailableRoutes({
       ...params,
-      apiUrl: this.apiUrl,
-      logger: this.logger,
+      apiUrl: params?.apiUrl ?? this.apiUrl,
```

Comment:
NIT: Wouldn't it make more sense to use `||` here? AFAICT with the current implementation, if I pass in an empty string `""` it would allow that

Reply:
Yeah that makes sense, would be good to provide as many guard rails as possible. 👍